### DISCUSSION TITLE
SMPTE ST 2110-41

### DISCUSSION THREAD

**Discussion by jtc-dolby** [vsf-tv/gccg-api#33] [General]
Regarding Metadata, I see the auxillary data format ST291 is included but not ST2110-41. Is this because there is an IANA Media Type for ST291 but not ST2110-41? If this was rectified, would it be reasonable to expect ST 2110-41 to be included? Would you need support from broadcasters?
James Cowdery - Dolby

**Comment by jmailhot** [comment]
agree it would be ideal to generically carry the -41 payload units in the same manner that we generically transport the ANC payload units.

**Reply by jtc-dolby** [reply]
Correcting my original post, there is already an IANA Media Type for ST 2110-41 so this a non-issue. Do you need a technical contribution from a VSF member? Can we expect the spec. to be enhanced to support ST 2110-41 in the future or do we need to contribute?
